frontend/packages/lib-components/src/molecules/FileDownloadButton.tsx

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import styled from 'styled-components'
+
+import { Result } from '@evaka/lib-common/src/api'
+import { downloadBlobAsFile } from '@evaka/lib-common/src/utils/file'
+import { Attachment } from '@evaka/lib-common/src/api-types/application/ApplicationDetails'
+import { espooBrandColors } from '@evaka/lib-components/src/colors'
+import { UUID } from '@evaka/lib-common/src/types'
+
+const DownloadButton = styled.button`
+  border: none;
+  text-align: start;
+  padding: 0;
+  background: none;
+  color: ${espooBrandColors.espooTurquoise};
```

Comment:
This color should be `blueColors.medium` if the idea is to make it look like an `a` element.

Reply:
:+1: Not sure where I picked this one from.